libres/lib/analysis/ies/ies_data.cpp

```diff
@@ -119,19 +103,18 @@ void ies::data::Data::augment_initialE(const matrix_type *E0) {
         int ens_size_msk = this->ens_mask_size();
         int m = 0;
         for (int iobs = 0; iobs < obs_size_msk; iobs++) {
-            if (!bool_vector_iget(this->m_obs_mask0, iobs) &&
-                bool_vector_iget(this->m_obs_mask, iobs)) {
+            if (!(this->m_obs_mask0[iobs]) && (this->m_obs_mask[iobs])) {
```

Comment:
The use of parenthesis around a single expression makes it harder to read - in my opinion.